test/LexerTest.php

```diff
@@ -39,126 +39,176 @@ public function testLexer($rql, $expected)
     public function lexerProvider()
     {
         return array(
-            'complex eq search concatenated fields' => array('eq(name-part+test,12)', array(
```

Comment:
It's still a case of this:

```php
            'out tests' => array('out(name,[foo,bar]', array(
```
vs. this
```php
            'out tests' => array(
                'out(name,[foo,bar]',
```
You should really tell your IDE to not change unrelated stuff to what you're working on or start staging your changes on a line-by-line base.
It's not like I'm against this kind of cleanup, but if it is being done in an automated fashion it should be on its own branch so I don't need to go through each commit of a PR to get what really changed.